src/python/zquantum/core/bitstring_distribution/distance_measures/mmd.py

```diff
@@ -51,19 +51,21 @@ def compute_rbf_kernel(x_i: np.ndarray, y_j: np.ndarray, sigma: float) -> np.nda
     return kernel_matrix
 
 
-def compute_multi_rbf_kernel(x_i: np.ndarray, y_j: np.ndarray, sigmas: List) -> np.ndarray:
-    """ Compute the multi-gaussian (RBF) kernel matrix K, with K_ij = 1/N * Sum_n [exp(-gamma_n |x_i - y_j|^2)] with n = 1,...,N and gamma = 1/(2*sigma).
+def compute_multi_rbf_kernel(
+    x_i: np.ndarray, y_j: np.ndarray, sigmas: List
+) -> np.ndarray:
+    """Compute the multi-gaussian (RBF) kernel matrix K, with K_ij = 1/N * Sum_n [exp(-gamma_n |x_i - y_j|^2)] with n = 1,...,N and gamma = 1/(2*sigma).
 
-        Args:
-            x_i (np.array): Samples A (integers).
-            y_j (np.array): Samples B (integers).
-            sigmas (np.array): The list of bandwidths of the multi-gaussian kernel.
+    Args:
+        x_i (np.array): Samples A (integers).
+        y_j (np.array): Samples B (integers).
+        sigmas (np.array): The list of bandwidths of the multi-gaussian kernel.
```

Comment:
So the typing says `List` and docstring says `np.array`.
Actually, since you already have type in the definition, you can get rid of that in the docstring to avoid redundancy + it should be `np.ndarray` and not `np.array`.

Reply:
Fixed in 4a5f84a92a6587f48d6cb7e0ecd7c56343ba28af